### ISSUE TITLE
[Question] Preparation of dataset?

### ISSUE THREAD

**Issue by Rafeal23** [breizhn/DTLN#15]
Hi Nils,

I trained your model on speech & noise provided by DNS challenge, but it seems that the model you provided in pretrained_model/ folder performs better than the model I trained (especially for the data with_reverb). Therefore I got some question about how you prepared your data and trained your model.

1) In your paper you mentioned that WHAMR corpus was also used, did you use it as train & cross validation & test or only test set?
2) Did you use DNS-Challenge script provided by microsoft to create the training set (I think they did not consider room impulse response in thier script)? Or did you add any RIR to create your training set?
3) In the DNS-Challenge repo you forked, I think you just randomly split the noisy data into train and val, which means that same speaker(s) may appear in both train and val, right?
4) Compared with DTLN_norm_500h.h5, did you use norm_stft=false and other parameters unchanged to get the model.h5 in pretrained_model?

**Comment by breizhn** [comment]
Hi Eric,

 > 1. In your paper you mentioned that WHAMR corpus was also used, did you use it as train & cross validation & test or only test set?

I didn’t train on this set and just used it as test set to show the performance and reverberated data.

> 2. Did you use DNS-Challenge script provided by microsoft to create the training set (I think they did not consider room impulse response in thier script)? Or did you add any RIR to create your training set?

Yes, I only used the DNS-Challenge data and trained only on dry data. I changed the configuration a bit. For this see the forked repo.

> 3. In the DNS-Challenge repo you forked, I think you just randomly split the noisy data into train and val, which means that same speaker(s) may appear in both train and val, right?

Yes this is correct.

> 4. Compared with DTLN_norm_500h.h5, did you use norm_stft=false and other parameters unchanged to get the model.h5 in pretrained_model?

Yes, also correct. The model.h5 is the same as handed in to the DNS-Challenge and described in the paper. There is no normalization of the STFT features performed in this model during training or testing. All other parameters were unchanged compared to the norm model.

**Comment by Rafeal23** [comment]
Hi Nils,
Thank you so much for your prompt reply. I understand much better now.
